specification/appconfiguration/data-plane/Microsoft.AppConfiguration/stable/2023-11-01/appconfiguration.json

```diff
@@ -2391,7 +2391,8 @@
             "in": "path",
             "description": "The name of the key-value snapshot to update.",
             "required": true,
-            "type": "string"
+            "type": "string",
+            "maxLength": 256
```

Comment:
@avanigupta I'm doing this cleanup to streamline the spec and make it easier to maintain. As part of using the same snapshot name definition the maxLength constraint was added over other instances that didnt previously specify it. I believe this change is correct given that it all refers to the snapshot name path param, but could you please confirm?

Reply:
On the service side, we only enforce the max length during PUT operation. But it should be fine to specify it for all operations with snapshot name in the API spec. 

This made me realize we have the same constraints for key, label, tagname and tagvalue parameters too. All of those are limited to 256 characters. Could you add the same maxLength property for all those params too? Or maybe this is something we need to do in a separate PR?

Reply:
Will create a separate PR to add missing properties

Reply:
sounds good, I'll leave the updated maxLength in this PR and let you make the updates to the other properties in a separate PR. Thanks for the info :)